.jenkins/pipelines/Azure/Linux/Jenkinsfile

```diff
@@ -229,6 +229,17 @@ def ACCHostVerificationTest(String version, String build_type) {
                            cmake ${WORKSPACE} -G Ninja -DBUILD_ENCLAVES=OFF -DHAS_QUOTE_PROVIDER=OFF -DCMAKE_BUILD_TYPE=${build_type} -Wdev
                            ninja -v
                            ctest -R host_verify --output-on-failure --timeout ${CTEST_TIMEOUT_SECONDS}
+                           cp tests/host_verify/host/*.der ${WORKSPACE}/samples/host_verify
+                           cp tests/host_verify/host/*.bin ${WORKSPACE}/samples/host_verify
+                           pushd ${WORKSPACE}/samples/host_verify
+                           mkdir build
+                           cd build
+                           cmake .. -G Ninja -DBUILD_ENCLAVES=OFF -DHAS_QUOTE_PROVIDER=OFF -DCMAKE_BUILD_TYPE=${build_type} -Wdev
```

Comment:
You can remove the `-DHAS_QUOTE_PROVIDER=OFF`. That option is no longer valid.

Reply:
In both cases?

Reply:
In all cases, yes.